.eslintrc.json

```
@@ -0,0 +1,22 @@
+{
+  "root": true,
+  "plugins": ["import", "sonarjs"],
+  "parserOptions": {
+    "ecmaVersion": 2020
+  },
+  "extends": ["airbnb-base", "plugin:sonarjs/recommended", "prettier"],
+  "rules": {
+    "sonarjs/cognitive-complexity": "warn"
```

We want file extension as this is a esm package

```suggestion
    "sonarjs/cognitive-complexity": "warn",
    "import/extensions": [
      "error",
      "ignorePackages",
      {
        "js": "always"
      }
    ]
```